### ISSUE TITLE
requestOptions documentation outdated

### ISSUE THREAD

**Issue by Mcayear** [AidanWelch/google-translate-api#22]
[google-translate-api#proxy](https://github.com/AidanWelch/google-translate-api#proxy)

code:
```
const tunnel = require('tunnel');
const translate = require('google-translate-api-x');

translate('Ik spreek Engels', { to: 'en', tld: 'jp' }, {
    agent: tunnel.httpsOverHttp({
        proxy: {
            host: '127.0.0.1', // ip
            port: '7890', // port
        }
    }
    )
}).then(res => {
    console.log(res)
    // do something
});
```

console:
```
PS C:\Users\Administrator\Desktop\NodejsProjects> node .
node:internal/deps/undici/undici:14062
    Error.captureStackTrace(err, this);
          ^

TypeError: fetch failed
    at Object.fetch (node:internal/deps/undici/undici:14062:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async runBatch (C:\Users\Administrator\Desktop\NodejsProjects\node_modules\google-translate-api-x\lib\translate.cjs:7:19) {
  cause: ConnectTimeoutError: Connect Timeout Error
      at onConnectTimeout (node:internal/deps/undici/undici:7897:28)
      at node:internal/deps/undici/undici:7855:50
      at Immediate._onImmediate (node:internal/deps/undici/undici:7884:37)
      at process.processImmediate (node:internal/timers:471:21) {
    code: 'UND_ERR_CONNECT_TIMEOUT'
  }
}

Node.js v18.13.0
```

**Comment by fatcarter** [comment]
The function in the `lib/translate.cjs` file takes only two arguments   :(

**Comment by AidanWelch** [comment]
> The function in the `lib/translate.cjs` file takes only two arguments   :(

Yeah it's an outdated thing in the documentation, you pass request options as the requestOptions property to the options object.

**Comment by AidanWelch** [comment]
Fixed in df898fd22039cffe08e413547461735e26d8f538
